projects/core/src/routing/facade/routing.service.spec.ts

```diff
@@ -174,6 +174,7 @@ describe('RoutingService', () => {
     );
   });
 
+  /*
```

Comment:
Please remove these tests, those are not needed anymore.

Reply:
Removed